src/shapes/triangle.py

```diff
@@ -0,0 +1,50 @@
+import cv2
+import numpy as np
+
+from src.shapes.point import Point
+from src.shapes.shape import Shape
+from src.shapes.line import Line
+from src.transform_shapes.transform_shapes import TransformShape
+
+
+class Triangle(Shape, TransformShape):
+    def __init__(self, point1: Point, point2: Point, point3: Point, line_color, image):
+        self.point1 = point1
+        self.point2 = point2
+        self.point3 = point3
+        self.line_color = line_color
+        self.image = image
+        center_x = (self.point1.x + self.point2.x + self.point3.x) / 3
+        center_y = (self.point1.y + self.point2.y + self.point3.y) / 3
+        self.center = Point(center_x, center_y, line_color, image)
+        self.line1, self.line2, self.line3 = self.create_lines_from_points()
+
+    def create_lines_from_points(self):
```

Comment:
this method should be private since it is only called inside the class. And it is better to compute it lazily, only at the moment of drawing create the lines and draw them